configs/components/cpp-pcp-client.rb

```diff
@@ -1,10 +1,14 @@
 component "cpp-pcp-client" do |pkg, settings, platform|
   pkg.load_from_json('configs/components/cpp-pcp-client.json')
+
   cmake = "/opt/pl-build-tools/bin/cmake"
   toolchain = "-DCMAKE_TOOLCHAIN_FILE=/opt/pl-build-tools/pl-build-toolchain.cmake"
-  pkg.environment "PATH" => "#{settings[:bindir]}:/opt/pl-build-tools/bin:$$PATH"
+  make = platform[:make]
+
+  unless platform.is_windows?
+    pkg.environment "PATH" => "#{settings[:bindir]}:/opt/pl-build-tools/bin:$$PATH"
+  end
```

Comment:
What do you mean? I'm setting path for windows on line 32, but I can move it here if that's easier to read?